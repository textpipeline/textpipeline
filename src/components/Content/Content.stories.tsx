import { storiesOf } from '@storybook/react';
import React from 'react';

import Content from './Content';

storiesOf('Content', module).add('default', () => (
  <Content>
    <p>
      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentesque interdum at tortor eget eleifend. Praesent
      consequat pretium fermentum. Cras sodales nec orci sed tincidunt. Pellentesque consectetur velit ac ligula
      facilisis, quis dapibus enim pulvinar. Integer aliquet ac tellus eu gravida.
    </p>
  </Content>
));
