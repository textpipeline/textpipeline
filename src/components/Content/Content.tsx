import Box from '@material-ui/core/Box';
import { createStyles, makeStyles } from '@material-ui/core/styles';
import React from 'react';

const contentMaxWidth = 1200;

const useStyles = makeStyles(() =>
  createStyles({
    main: {
      width: '100%',
    },
  })
);

const Content: React.FC = ({ children }) => {
  const classes = useStyles();
  return (
    <main className={classes.main}>
      <Box p={3} width="100%" height="100%" maxWidth={contentMaxWidth} mx="auto">
        {children}
      </Box>
    </main>
  );
};

export default Content;
