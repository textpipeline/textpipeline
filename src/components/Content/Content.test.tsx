import { shallow, ShallowWrapper } from 'enzyme';
import React from 'react';

import Content from './Content';

describe('Content', () => {
  let rendered: ShallowWrapper;

  beforeEach(() => {
    rendered = shallow(<Content>stuff</Content>);
  });

  it('renders correctly', () => {
    expect(rendered).toMatchSnapshot();
  });
});
