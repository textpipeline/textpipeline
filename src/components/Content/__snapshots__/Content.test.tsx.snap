// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content renders correctly 1`] = `
<main
  className="makeStyles-main-1"
>
  <Styled(MuiBox)
    height="100%"
    maxWidth={1200}
    mx="auto"
    p={3}
    width="100%"
  >
    stuff
  </Styled(MuiBox)>
</main>
`;
